test/exfmt/integration/typespec_test.exs

```diff
@@ -36,18 +36,16 @@ defmodule Exfmt.Integration.TypespecTest do
     """
     # left side too long
     assert_format """
-    @spec run(String.t,
-              term,
-              [meta],
-              options)
+    @spec run(
+        String.t, term, [meta], options
```

Comment:
I don't think these should be all on one line if the call has been split

Reply:
I agree. I'll update this PR with a fix for this shortly.